storage-model/src/test/java/no/unit/nva/publication/storage/model/daos/DaoUtils.java

```diff
@@ -27,20 +29,23 @@
 public final class DaoUtils {
 
     private static final Clock clock = Clock.systemDefaultZone();
-    public static final ResourceTest resourceGenerator = new ResourceTest();
 
-    public static Resource sampleResource() throws InvalidIssnException, MalformedURLException {
-        return Resource.fromPublication(resourceGenerator.samplePublication(
-            resourceGenerator.sampleJournalArticleReference()));
+
+    public static Resource sampleResource() {
+        Publication publication = PublicationGenerator.randomPublication();
+        var additionalIdentifier = Set.of(new AdditionalIdentifier(CRISTIN_SOURCE, randomString()));
```

Comment:
We need to add a Cristin identifier because we use the sampleResource for Cristin tests